cobigen-eclipse/cobigen-eclipse/src/com/devonfw/cobigen/eclipse/generator/GeneratorWrapperFactory.java

```diff
@@ -206,13 +209,19 @@ else if (selection instanceof IStructuredSelection) {
      * @throws GeneratorCreationException
      *             if the generator configuration project does not exist
      */
-    private static CobiGen initializeGenerator()
-        throws GeneratorProjectNotExistentException, InvalidConfigurationException, GeneratorCreationException {
+    private static CobiGen initializeGenerator() throws InvalidConfigurationException, GeneratorCreationException {
 
         try {
             ResourcesPluginUtil.refreshConfigurationProject();
             IProject generatorProj = ResourcesPluginUtil.getGeneratorConfigurationProject();
-            return CobiGenFactory.create(generatorProj.getLocationURI());
+            if (null == generatorProj.getLocationURI()) {
+                String fileName = ResourcesPluginUtil.downloadJar(false);
+                IPath ws = ResourcesPluginUtil.getWorkspaceLocation();
+                File file = new File(ws.append(ResourceConstants.DOWNLOADED_JAR_FOLDER + "\\" + fileName).toString());
```

Comment:
From Malte's comment on the other PR:
>Make sure, it is linux compliant as well -> path separator!

So instead of the `"\\"` you will have to use a correct path separator (see https://stackoverflow.com/questions/3548775/platform-independent-paths-in-java)